src/api/firebase.js

```diff
@@ -42,6 +42,13 @@ export function getItemData(snapshot) {
 
 		return data;
 	});
+
+	// filter the data from firebase to remove the hidden placeholder value
+	const filteredArray = arrayFromFirestore.filter(
+		(doc) => doc.hidden === false,
+	);
+
+	return filteredArray;
```

Comment:
```suggestion

```
since we are no longer adding the place hold item it isn't necessary to filter the data anymore, which added more complexity.